app/views/user_mailer/room_confirm.html.erb

```diff
@@ -0,0 +1,2 @@
+<h2>Room confirmation for <%= @user.name %> </h2>
+<p>Thank you for using Schedlr! Your reservation has been created for <%= @event %>
```

Comment:
When using simply `@event` we will have to override the `.to_s` method on `Event` otherwise it'll just print `<Event#298343>`